.github/workflows/ci.yaml

```diff
@@ -27,6 +27,7 @@ jobs:
       CXXFLAGS: "-Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls -Wno-deprecated-copy"
       DOCKER_IMAGE: ghcr.io/ros-planning/moveit2:${{ matrix.env.IMAGE }}
       UPSTREAM_WORKSPACE: moveit2.repos $(f="moveit2_$(sed 's/-.*$//' <<< "${{ matrix.env.IMAGE }}").repos"; test -r $f && echo $f)
+      TARGET_WORKSPACE: $TARGET_REPO_PATH github:henningkayser/moveit_resources#pr-port_prbt_packages
```

Comment:
Shouldn't you drop this change now that it is merged into moveit_resources?

Reply:
I thought I just did this... I must have failed to commit it

Reply:
Should be fixed now. We still need moveit_resources in the target workspace because of the circular dependency (see #885). It's the same issue with MoveIt 1